README.md

```diff
@@ -1,3 +1,15 @@
+![Leanplum](Leanplum.png)
+
+<p align="center">
+<img src='https://jenkins.leanplum.com/buildStatus/icon?job=apple-sdk-github'></a>
```

Comment:
I don't think we should add this here. The badges are public right? I think apple-sdk-github is used for PR so it can give a false alarm.